integration_tests/integration/prison-register-details.spec.ts

```diff
@@ -26,6 +26,8 @@ context('Prison register - prison details navigation', () => {
     prisonDetailsPage.prisonDetailsSection().should('contain.text', 'Open')
     prisonDetailsPage.prisonDetailsSection().should('contain.text', 'Male prison')
     prisonDetailsPage.prisonDetailsSection().should('contain.text', 'Female prison')
+    prisonDetailsPage.prisonDetailsSection().should('contain.text', 'Her Majesty’s Prison,')
```

Comment:
Would've prefer to check for `Her Majesty’s Prison, Her Majesty’s Youth Offender Institution` but think I ran into issues with the size of the Cypress window so the test continually failed